common/tools/dev-tool/src/commands/samples/run.ts

```diff
@@ -26,10 +26,10 @@ export const commandInfo = makeCommandInfo(
 async function runSingle(name: string, accumulatedErrors: Array<[string, string]>) {
   log("Running", name);
   try {
-    if (/.*\/samples\/.*/.exec(name)) {
+    if (/.*[\\\/]samples[\\\/].*/.exec(name)) {
```

Comment:
@willmtemple can you review the changes to this dev-tool file?

Reply:
For context, running samples failed for me on Windows because the paths had backslashes.

Reply:
yeah it seems reasonable, if `name` is a path that isn't normalized, though maybe we should file an enhancement for doing that

Reply:
It's gone through some full-path-ification and gotten \ everywhere. I didn't study it closely, though.

Reply:
Fine by me! I did believe that node module paths were defined to use forward slashes on windows as well, but maybe because these aren't package imports that doesn't apply.

Reply:
@willmtemple indicated he is fine with this offline.